search-api/openshift/templates/dc.json

```diff
@@ -0,0 +1,353 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "annotations": {
+      "description": "Deployment template for a auth api service.",
+      "tags": "${NAME}-${TAG_NAME}"
+    },
+    "name": "${NAME}-${TAG_NAME}-deploy"
+  },
+  "objects": [
+    {
+      "kind": "DeploymentConfig",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-${TAG_NAME}",
+        "labels": {
+          "app": "${NAME}-${TAG_NAME}",
+          "app-group": "${APP_GROUP}",
+          "template": "${NAME}-deploy"
+        }
+      },
+      "spec": {
+        "strategy": {
+          "type": "Rolling",
+          "rollingParams": {
+            "updatePeriodSeconds": 1,
+            "intervalSeconds": 1,
+            "timeoutSeconds": 600,
+            "maxUnavailable": "25%",
+            "maxSurge": "25%",
+            "pre": {
+              "failurePolicy": "Abort",
+              "execNewPod": {
+                "command": ["/opt/app-root/src/pre-hook-update-db.sh"],
```

Comment:
We don't want to run this, we don't have our own DB. Probably the lear/colin-api example is better.